Specs/DateTime/Chinese/DateExtractor.json

```diff
@@ -144,9 +144,10 @@
   },
   {
     "Input": "10月12号，星期一",
+    "NotSupported": "java, javascript, python",
```

Comment:
Is this just a matter of making sure normalization always happen in the other plats too?

Reply:
Since preprocessing does not happens in the other platforms, these test cases now fail because the extracted text is not normalized.

Reply:
added comment